tools/libfrencutils/create_xgrid.c

```diff
@@ -887,6 +885,174 @@ int create_xgrid_2dx2d_order2_(const int *nlon_in, const int *nlat_in, const int
 
 };
 #endif
+#ifdef _OPENACC
```

Comment:
do we need an _openacc  macro since the directives will be ignored without the -acc compiler flag?

Reply:
Yeah it doesn't check for compiling its more for whats in the `#else`. It uses it to figure out if it should use the openmp or openacc version of the `create_xgrid_2dx2d_order2` function.

With this logic it'll also default to compiling the old openmp version if neither openmp or acc is enabled which I think is what we want.